README.md

```diff
@@ -1,85 +1,67 @@
-# Membership
-
-The Node.js Foundation has an Individual Membership Program.
-
-The Foundation is a member based organization. Previously the
-foundation only had the ability for companies and non-profits
-to be members, but as of November 2015, the Foundation also
-accepts [individual humans as members][4].
-
-Individual members of the Node Foundation have the ability to
-both run and vote for representation on the [Board of Directors][5].
-
-## 2017 Election
-
-Nominations for the 2017 Election were posted [here][11].
-
-The winner of the 2017 election will serve a two year term. They will join 
-Ashley Williams [@ashleygwilliams](https://github.com/ashleygwilliams) who will
-serve with the elected director in 2017 for the balance of the 2 year term.
-
-For more information about Individual Membership on the Board,
-see below.
-
-## 2016 Election
-
-Nominations for the 2016 Election were posted [here][1].
-
-The 2016 election was the first Individual Membership Election. The winner of the 
-election is serving a two year term and the runner-up is serving a one year term.
-Future elections will happen each year for a new two year term.
-
-- The board members elected in the 2016 election are:
-  - Ashley Williams [@ashleygwilliams](https://github.com/ashleygwilliams) for a 2 year term.
-  - Feross Aboukhadijeh [@feross](https://github.com/feross) for a 1 year term.
-
-## Cost
-
-- Node.JS Project Contributors (any member of the GitHub Org): [__free__][2]*
+# Board of Directors
+
+The Node.js Foundation is a member based organization. There are 5 classes of Membership: 
+Platinum, Gold, and Silver Members, Associated Entity Members and Individual Members.
+
+A list of non-Individual members and additional information is available on the website:
+https://nodejs.org/en/foundation/members/
+
+## Individual Members
+Individual Members of the Node.js Foundation have the ability to both run and vote for 
+representation on the [Board of Directors](https://nodejs.org/en/foundation/board/).
+
+### Representation on the Board of Directors
+The Individual Members are represented on the Board of Directors by 2 Individual Directors 
+with staggared two year terms. A representivite is elected in January of each year with
+no limitations to the number of terms served.
+
+#### Current Individual Directors
+<!-- Senior Director always listed first. Upon re-election, they become the Junior again.-->
```

Comment:
As it is just a policy I made up (rather than an enforceable rule) & I'd like to strive for... I was trying to be subtle. I don't know how we would want to state things like this. Can you do a PR?